charon/src/translate/translate_functions_to_ullbc.rs

```diff
@@ -1462,16 +1463,21 @@ impl<'tcx, 'ctx, 'ctx1> BodyTransCtx<'tcx, 'ctx, 'ctx1> {
         mut self,
         rust_id: DefId,
         arg_count: usize,
+        item_meta: &ItemMeta,
     ) -> Result<Option<ExprBody>, Error> {
         // Stopgap measure because there are still many panics in charon and hax.
-        let mut this = panic::AssertUnwindSafe(&mut self);
-        let res = panic::catch_unwind(move || this.translate_body_aux(rust_id, arg_count));
-        match res {
-            Ok(Ok(body)) => Ok(body),
-            Ok(Err(e)) => Err(e),
-            Err(_) => {
-                let span = self.t_ctx.tcx.def_span(rust_id);
-                error_or_panic!(self, span, "Thread panicked when extracting body.");
+        if item_meta.opaque {
```

Comment:
I wrote we should put this with the other checks for opacity in `translate_body` but it was actually `translate_body_aux` (see the checks starting at line 1493).